apps/jacket/jacket.star

```diff
@@ -182,60 +186,53 @@ def get_weather_data(config):
         if api_key == None:
             print("Missing api_key")
             return SAMPLE_STATION_RESPONSE
-        if application_key == None:
-            print("Missing application_key")
+        if location == None:
+            print("Missing location")
             return SAMPLE_STATION_RESPONSE
 
         print("Getting new weather data")
+        location = json.decode(location)
+        query = "%s?exclude=minutely,hourly,daily,alerts&lat=%s&lon=%s&appid=%s" % (OPEN_WEATHER_URL, location["lat"], location["lng"], api_key)
         res = http.get(
```

Comment:
Note that the http module now has caching built in. You can pass `ttl_seconds` to `http.get()` and it'll all be handled for you!

Reply:
@matslina this is updated